cmake/projects/PocoCpp/hunter.cmake

```diff
@@ -8,11 +8,11 @@ hunter_add_version(
     PACKAGE_NAME
     PocoCpp
     VERSION
-    1.7.3
+    1.7.4
```

Comment:
Please **add** new version, do not erase old. See example: https://github.com/ruslo/hunter/blob/master/cmake/projects/GTest/hunter.cmake. Thanks.

More details:
* https://docs.hunter.sh/en/latest/overview/what-is-it.html
* https://docs.hunter.sh/en/latest/overview/customization/config-id.html
* https://docs.hunter.sh/en/latest/overview/customization/hunter-id.html